bundles/org.openhab.core.io.net/src/main/java/org/openhab/core/io/net/http/internal/WebClientFactoryImpl.java

```diff
@@ -124,16 +125,28 @@ protected void deactivate() {
 
     @Override
     public HttpClient createHttpClient(String consumerName) {
+        return createHttpClient(consumerName, null);
+    }
+
+    @Override
+    public @NonNull HttpClient createHttpClient(@NonNull String consumerName,
```

Comment:
Why do you add `@NonNull` here? The class (and interface) is annotated with `@NonNullByDefault`, so it should not be necessary.

Reply:
A copy&paste from the same file, I believe, and I even do not see these annotations. I will of course remove them.

Reply:
Fixed.
It was not a copy&paste but Eclipse that generates this method prototype.